content/en/docs/collector/custom-collector.md

```diff
@@ -59,12 +59,12 @@ Here are the tags for the `dist` map:
 | ---------------- | -------------------------------------------------------------------------------------------------- | -------- | --------------------------------------------------------------------------------- |
-| module:          | The module name for the new distribution, following Go mod conventions. Optional, but recommended. | Yes      | "go.opentelemetry.io/collector/cmd/builder"                                       |
-| name:            | The binary name for your distribution                                                              | Yes      | "otelcol-custom"                                                                  |
-| description:     | A long name for the application.                                                                   | Yes      | "Custom OpenTelemetry Collector distribution"                                     |
-| otelcol_version: | The OpenTelemetry Collector version to use as base for the distribution.                           | Yes      | "{{% param collectorVersion %}}"                                                                          |
-| output_path:     | The path to write the output (sources and binary).                                                 | Yes      | "/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831" |
-| version:         | The version for your custom OpenTelemetry Collector.                                               | Yes      | "1.0.0"                                                                           |
+| module:          | The module name for the new distribution, following Go mod conventions. Optional, but recommended. | Yes      | `go.opentelemetry.io/collector/cmd/builder`                                       |
+| name:            | The binary name for your distribution                                                              | Yes      | `otelcol-custom`                                                                  |
+| description:     | A long name for the application.                                                                   | Yes      | `Custom OpenTelemetry Collector distribution`                                     |
+| otelcol_version: | The OpenTelemetry Collector version to use as base for the distribution.                           | Yes      | `{{% param collectorVersion %}}`                                                  |
+| output_path:     | The path to write the output (sources and binary).                                                 | Yes      | `/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831` |
+| version:         | The version for your custom OpenTelemetry Collector.                                               | Yes      | `1.0.0`                                                                           |
```

Comment:
@cartermp et al., note the changes in the **Default value** column: I dropped the quotes and went for code fence syntax. This feels slightly more accurate. (In particular because the YAML syntax doesn't require quotes for those entries -- in particular version IDs.) WDYT?

Reply:
I like it